tests/CMakeLists.txt

```diff
@@ -81,6 +80,11 @@ if (UNIX OR ADD_WINDOWS_ENCLAVE_TESTS OR USE_CLANGW)
         add_subdirectory(thread_local_no_tdata)
         add_subdirectory(VectorException)
         add_subdirectory(stack_smashing_protector)
+
+        # Skip the following tests when enabling code coverage tests.
```

Comment:
Why do we need to skip this directory?

Reply:
The reason is that the libgcov relies on host fs to write runtime data to the file upon the enclave termination. Tests in this directory (and also child_process) have the pattern of multiple invocations of `oe_terminate_enclave`. The first invocation will free the data structures associated to host fs and the writeout operations in the subsequent invocations will fail.

I've added comments to explain the reason.